examples/custom-colors/index.html

```diff
@@ -30,7 +30,7 @@
         <section class="navbar-section">
             <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
             <div class="dropdown mr-2">
-                <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0" onclick="return false;">
```

Comment:
@aibcmars if you know a better way please commit directly in this PR

Reply:
yes, I have replaced anchor with span - anchor with href="#" is always going to the top
I have also changed the dropdown display direction so it is nicer(not cut)